main.js

```diff
@@ -3,10 +3,32 @@ var reservations = {
   'Ted': { claimed: true }
 }
 
-var name = prompt('Please enter the name for your reservation');
-
-var claimReservation = function () {
-  // write your code here!
+let namesArray=Object.keys(reservations)
+let c=0;
+var claimReservation= function(input){
+let name= document.getElementById("input").value;
+  for(let i of namesArray){
+  name = name.toLowerCase();
+  var x=i.toLowerCase();
+    if (name==x){
+      if(reservations[i].claimed==false){
+      document.getElementById("live").innerHTML = (i+" welcome");
+      reservations[i].claimed==true
+      }
+      else if(reservations[i].claimed==true){
+      document.getElementById("live").innerHTML = (i+" someone has alrady claimed your reservation");
```

Comment:
DRY
Don't Repeat Yourself
since you're doing `document.getElementById("live") twice - better to store it in a variable and use it both times